test/functionalTest/cases/3004_v2_reg_create/v2_reg_create.test

```diff
@@ -97,7 +97,7 @@ Date: REGEX(.*)
 02. GET the registration using APIv2
 ====================================
 HTTP/1.1 200 OK
-Content-Length: 312
```

Comment:
Content length changes in this PR are due to httpInfo rendering has moved from `http":{"url": "http://foo.bar"}` to `http":{"url":"http://foo.bar"}` (1 less whitespace). Thus, the change in .test is fine.

NTC